net/unbound/files/unbound.init

```diff
@@ -1,20 +1,38 @@
 #!/bin/sh /etc/rc.common
-# Copyright (C) 2016 Michael Hanselmann
-
-START=61
+##############################################################################
+#
+# Copyright (C) 2016 Michael Hanselmann, Eric Luehrsen
+#
+##############################################################################
+#
+# This init script is just the entry point for Unbound UCI.
+#
+##############################################################################
 
+START=60
 USE_PROCD=1
+PROG="/usr/sbin/unbound"
+
+##############################################################################
+
+. /usr/lib/unbound/unbound.sh
```

Comment:
I would not be so absolute. _Most_ commits are delta function in an otherwise prepared project. Yes, each one should be whole and functioning. It can be necessary to be more pragmatic to difference heuristics. Only one example file, directory, or container rename must be separate from file content that updates references (any version management system seems to have this hick up).

I intend to reorder and squash, but for now its easier to "fixup" with smaller broken pieces. I am polishing the full UCI. I am trying to figure the best way to publish this intermediate step (and walk the right pieces back and forth).